Tests/Helper/DateTimeHelperTest.php

```diff
@@ -117,4 +117,25 @@ public function testLocaleTimezones()
         $this->assertEquals('12:37', $helper->format($dateParis, 'HH:mm'), 'A date in the Europe/Paris timezone, should be corrected when formatted with timezone Europe/London.');
         $this->assertEquals('13:37', $helperWithMapping->format($dateParis, 'HH:mm'), 'A date in the Europe/Paris timezone, should be corrected when formatted with timezone Europe/Paris.');
     }
+
+    public function testImmutable()
+    {
+        if (!class_exists('\DateTimeImmutable')) {
```

Comment:
Use `@requires PHP 5.5` on the test method phpdoc instead.

See: https://phpunit.de/manual/current/en/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests.skipping-tests-using-requires